packages/admin-app/tsconfig.json

```diff
@@ -1,7 +1,21 @@
 {
-    "extends": "../../tsconfig",
     "compilerOptions": {
-        "jsx": "react-jsx"
+        "module": "esnext",
+        "lib": ["dom", "dom.iterable", "esnext"],
+        "moduleResolution": "node",
+        "jsx": "react-jsx",
+        "allowJs": true,
+        "esModuleInterop": true,
+        "skipLibCheck": true,
+        "allowSyntheticDefaultImports": true,
+        "strict": true,
+        "forceConsistentCasingInFileNames": true,
+        "noFallthroughCasesInSwitch": true,
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "noEmit": true,
+        "target": "es5"
```

Comment:
For both React projects, the target is set to es5, but in the lib section esnext is used.  Are these being different expected?